Tests/PackageGraphTests/PubgrubTests.swift

```diff
@@ -770,11 +760,20 @@ final class PubgrubTests: XCTestCase {
         XCTAssertEqual(unavailable.description, "{package 1.0.0}")
     }
 
-    func DISABLED_testNonExistentPackage() {
+    func testNonExistentPackage() {
         builder.serve(root: "root", with: ["package": .exact(v1)])
 
         let resolver = builder.create()
-        _ = resolver.solve(root: rootRef, pins: [])
+        let result = resolver.solve(root: rootRef, pins: [])
+
+        guard
+            case .error(let error) = result,
+            let anyError = error as? AnyError,
+            let mockError = anyError.underlyingError as? _MockLoadingError,
+            case .unknownModule = mockError
```

Comment:
What is unknown module? Shouldn't it be package-not-found or something?

Reply:
This was already included with the mock loading error, which only comes up in tests. For actual results it would throw whatever `getContainer(for identifier: PackageReference)` throws. Also, I don't actually know how to trigger this with an actual package. How would that even get into this case?